services/apps/portal/src/alcs/code/code.controller.spec.ts

```diff
@@ -44,7 +44,6 @@ describe('CodeController', () => {
         uuid: 'fake-uuid',
         name: 'fake-name',
         isFirstNation: false,
-        isActive: true,
```

Comment:
Is it not relevant anymore?

Reply:
This was a mistake by me, it was never actually sent over from ALCS, so it was always undefined.